tests/main.c

```diff
@@ -0,0 +1,33 @@
+#include "gtest_like_c.h"
+
+TEST(Aarch64MinGW, BigDataTest);
+TEST(Aarch64MinGW, CHKSTKTest);
+TEST(Aarch64MinGW, DllTest);
+TEST(Aarch64MinGW, MathTest);
+TEST(Aarch64MinGW, NestedFunction);
+TEST(Aarch64MinGW, OmpTest);
+TEST(Aarch64MinGW, TestUnwindStack);
+TEST(Aarch64MinGW, SJLJTest);
+TEST(Aarch64MinGW, StructTest);
+TEST(Aarch64MinGW, TestVaList);
+TEST(Aarch64MinGW, TestSPrintf);
+TEST(Aarch64MinGW, TestVaArgPack);
+
+int WinMain() {
```

Comment:
Why `WinMain` was used instead of the standard `main`?

Reply:
Changed.